js/app.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+const p4Req = new XMLHttpRequest();
+
+function p4ReqListener() {
+  const p4Obj = JSON.parse(this.responseText);
+  const p4NameHeader = document.querySelector('#person4Name');
+  p4NameHeader.innerHTML = p4Obj.name;
+
+  const p4HomeReq = new XMLHttpRequest();
+
+  function p4HomeReqListener() {
+    const p4Home = JSON.parse(this.responseText);
+    const p4HomeHeader = document.querySelector('#person4HomeWorld');
+    p4HomeHeader.innerHTML = p4Home.name;
+  }
+
+  p4HomeReq.addEventListener('load', p4HomeReqListener);
+  p4HomeReq.open('get', p4Obj.homeworld);
+  p4HomeReq.send();
+}
+
+p4Req.addEventListener('load', p4ReqListener);
```

Comment:
Nothing wrong with this but in general I'd like to see anything that has functionality to be written in a function that has descriptive naming and is invoked explicitly by an init() method as a good practice.